src/src/core/StringUtils.cpp

```diff
@@ -24,20 +24,37 @@ BEGIN_FTC_CORE
 namespace StringUtils
 {
 
+void ltrim(std::string &s)
+{
+    s.erase(s.begin(), std::find_if(s.begin(), s.end(), [](int ch) {
+        return !std::isspace(ch);
+    }));
+}
 void ltrim(std::string &s, char remove)
 {
     s.erase(s.begin(), std::find_if(s.begin(), s.end(), [remove](int ch) {
```

Comment:
char -> int promotion is redundant.

```suggestion
    s.erase(s.begin(), std::find_if(s.begin(), s.end(), [remove](char ch) {
```